src/script/webAudio.ts

```diff
@@ -63,9 +76,21 @@ export default class WebAudio {
   }
 
   static unmute(filter) {
-    if (filter.muted === true) {
-      filter.muted = false;
-      chrome.runtime.sendMessage({mute: false});
+    if (filter.muted) {
+      switch(filter.cfg.filterAudioMethod) {
+        case 0: // Mute tab
+          filter.muted = false;
+          chrome.runtime.sendMessage({mute: false});
+          break;
+        case 1: { // Mute video
+          let video = document.getElementsByTagName('video')[0];
+          if (video.volume == 0) {
```

Comment:
Could the volume be unintentionally turned on in the following scenario?

1. User has volume on
2. Profanity is muted and unmuted, leaving the volume in `filter.volume`.
3. User turns the volume to zero.
4. Profanity is muted, but `video.volume > 0` is false, so it does nothing on mute
5. Profanity unmutes `video.volume == 0` is true, so it restores the old `filter.volume`.

I'm not understanding the use of those two conditions.